roles/core/remote_mount/tasks/mount_filesystem.yml

```diff
@@ -0,0 +1,400 @@
+---
+#
+# Get some cluster information
+#
+- name: Storage Cluster (owner) | GET the Cluster Information
+  uri:
+    validate_certs: no
+    force_basic_auth: yes
+    url: https://{{ storage_cluster_ip }}:{{ gui_port }}/{{ scalemgmt_endpoint }}/cluster
+    method: GET
+    user: "{{ gui_user }}"
+    password: "{{ gui_password }}"
+    body_format: json
+    status_code:
+      - 200
+  register: owning_cluster_info
+
+- name: DEBUG | Storage Cluster | Print the Cluster Information
+  debug:
+    msg: "{{ owning_cluster_info }}"
+  when: debug is true
+
+- name: Client Cluster (access) | GET the Cluster Information
+  uri:
+    validate_certs: no
+    force_basic_auth: yes
+    url: https://{{ client_cluster_ip }}:{{ gui_port }}/{{ scalemgmt_endpoint }}/cluster
+    method: GET
+    user: "{{ gui_user }}"
+    password: "{{ gui_password }}"
+    body_format: json
+    status_code:
+      - 200
+  register: access_cluster_info
+
+- name: DEBUG | Client Cluster | Print the Cluster Information
+  debug:
+    msg: "{{ access_cluster_info }}"
+  when: debug is true
+
+- set_fact:
+    access_cluster_name: "{{ access_cluster_info.json.cluster.clusterSummary.clusterName }}"
+    owning_cluster_name: "{{ owning_cluster_info.json.cluster.clusterSummary.clusterName }}"
+
+- name: DEBUG | Print out the Storage and Client Cluster Names
+  debug:
+    msg: "Access: {{ access_cluster_name }} and Owning: {{ owning_cluster_name }}"
+  when: debug is true
+
+#
+# When forceRun, do some cleanup to properly unmount and remove things
+#
+- import_tasks: cleanup_filesystem.yml
+  when: forceRun | bool
```

Comment:
the `forceRun` is currently intended to be run by me during my Unit Testing to clean up the REST API calls so that I don't hit anything "already defined", things like that.   I invoked it from the CLI when running the role: `ansible-playbook playbook.yml -e "forceRun=true"`.  

You're right, calling the REST API twice will cause it to fail, this is why I implemented the forceRun, really for ease of testing purposes.  I don't expect the operator to ever call this.